opendalfs/file.py

```diff
@@ -31,22 +31,31 @@ def __init__(
             **kwargs,
         )
 
-    def _fetch_range(self, start: int, end: int):
-        """Download data between start and end"""
-        pass
+    def _fetch_range(self, start: int, end: int) -> bytes:
+        """Download data between start and end."""
+        logger.debug(f"Fetching bytes from {start} to {end} for {self.path}")
+        data = self.fs.fs.read(self.path)  # sync operator
```

Comment:
We now have native support for `read(self.path, offset, size)` after https://github.com/apache/opendal/pull/6086 was merged. Would you like to create an issue to track this, so we can update our usage and avoid reading the entire file?